src/cms/views/pages/page_tree_view.py

```diff
@@ -4,23 +4,17 @@
 from django.shortcuts import render, redirect
 from django.utils.decorators import method_decorator
 from django.utils.translation import ugettext as _
-
 from django.views.generic import TemplateView
 
+from ...constants import translation_status
 from ...decorators import region_permission_required
+from ...forms.pages.page_filter_form import PageFilterForm
 from ...models import Region, Language
```

Comment:
```suggestion
from ...models import Region, Language
from .page_mixin import PageMixin
```
The `PageMixin` is required.